pilot/pkg/model/telemetry.go

```diff
@@ -373,9 +374,33 @@ func (t *Telemetries) Tracing(proxy *Proxy) *TracingConfig {
 	return &cfg
 }
 
+var defaultConfigSource = &core.ConfigSource{
+	ConfigSourceSpecifier: &core.ConfigSource_Ads{
+		Ads: &core.AggregatedConfigSource{},
+	},
+	ResourceApiVersion:  core.ApiVersion_V3,
+	InitialFetchTimeout: &durationpb.Duration{Seconds: 0},
+}
+
 // HTTPFilters computes the HttpFilter for a given proxy/class
 func (t *Telemetries) HTTPFilters(proxy *Proxy, class networking.ListenerClass) []*hcm.HttpFilter {
 	if res := t.telemetryFilters(proxy, class, networking.ListenerProtocolHTTP); res != nil {
+		if features.EnableECDSForStats {
+			return []*hcm.HttpFilter{
+				{
+					Name: ECDSResourceName(networking.ListenerProtocolHTTP, class, proxy.Type),
+					ConfigType: &hcm.HttpFilter_ConfigDiscovery{
+						ConfigDiscovery: &core.ExtensionConfigSource{
+							ConfigSource: defaultConfigSource,
+							TypeUrls: []string{
+								xds.StatsFilterType,
+								xds.RBACHTTPFilterType,
```

Comment:
why rbac? that  was for WASM hacks, I don't think we need it here?

Reply:
Agree, please remove RBAC here.